model/proto/api_v2/sampling.proto

```diff
@@ -46,24 +46,6 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
   schemes: HTTPS;
 };
 
-message PostSpansRequest {
-    Batch batch = 1 [
-        (gogoproto.nullable) = false
-    ];
-}
-
-message PostSpansResponse {
-}
-
-service CollectorService {
-    rpc PostSpans(PostSpansRequest) returns (PostSpansResponse) {
-        option (google.api.http) = {
-            post: "/api/v2/spans"
-            body: "*"
-        };
-    }
-}
-
 enum SamplingStrategyType {
```

Comment:
Want to be careful with this one and test whether the older agent using grpc to pull sampling strategies would still work once the collector is upgraded to these changes. Otherwise we don't give a clean upgrade path to people.

I assume since we're not changing the actual proto namespace, it should work fine,  but best  to verify.